addons/test_mail/tests/test_performance.py

```diff
@@ -977,15 +932,14 @@ def test_complete_message_post(self):
         ]
         self.attachements = self.env['ir.attachment'].with_user(self.env.user).create(self.vals)
         attachement_ids = self.attachements.ids
-        with self.assertQueryCount(emp=76):
+        with self.assertQueryCount(emp=78):
```

Comment:
I probably asked the question already in my previous review and forgot the answer, but what are the 2 extra queries?
It's strange to have more queries even though we are removing a feature.

Reply:
Reading my comment I again, I guess that might be extra queries due to using `channel.partner` somewhere instead of writing on the now computed partner relation?